flycheck.el

```diff
@@ -2300,7 +2334,6 @@ Slots:
   '(
     ;; Handle events that may start automatic syntax checks
     (after-save-hook        . flycheck-handle-save)
-    (after-change-functions . flycheck-handle-change)
```

Comment:
I don't think we need to move this to the second hooks list.  This just triggers a syntax check on idle, if they are activated, but the displaying is controlled by `flycheck-finish-current-syntax-check` anyway.